lib/pf/profile/filter/day_of_week.pm

```diff
@@ -0,0 +1,101 @@
+package pf::profile::filter::day_of_week;
+=head1 NAME
+
+pf::profile::filter::day_of_week add documentation
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::profile::filter::day_of_week
+
+=cut
+
+use strict;
+use warnings;
+use Date::Format;
+use Moo;
+extends 'pf::profile::filter';
+
+=head1 ATTRIBUTES
+
+=head2 allowed_days
+
+The allowed days of the week
+
+=cut
+
+has allowed_days => ( is => 'rw' );
+
+=head2 value
+
+add a trigger to the value to create update allowed_days
+
+=cut
+
+has '+value' => ( trigger => 1 );
+
+=head1 METHODS
+
+=head2 match
```

Comment:
+1 for that documentation header.

This is a perfect example of something that needed additional info and that header covers it